fix(LIC): clarify how forwarding works (#11138)
newrelic/docs-website: src/content/docs/logs/logs-context/java-configure-logs-context-all.mdx

  >
    This is the simplest approach, and it's a great choice for developers who may not have the access or interest in setting up a log forwarder, or for accounts that want to see the power of logs and other linking metadata in context of their apps, without a lot of overhead.

>     Using this option, the agent adds `span.id`, `trace.id`, `hostname`, `entity.guid`, and `entity.name`. [Learn about log forwarding limitations.](/docs/logs/logs-context/get-started-logs-context#forwarding-details)

Reviewer: ```suggestion
    Using this option, the agent adds `span.id`, `trace.id`, `hostname`, `entity.guid`, and `entity.name`. [Learn about log forwarding limitations](/docs/logs/logs-context/get-started-logs-context#forwarding-details).
```

    This functionality is enabled by default.

>     All you need to do is install an agent version with log forwarding capabilities ([Java agent 7.6.0 or higher](/docs/release-notes/agent-release-notes/java-release-notes/)). If forwarding is disabled, you can use the following configuration.

Reviewer: ```suggestion
    All you need to do is install an agent version with log forwarding capabilities ([Java agent 7.6.0 or higher](/docs/release-notes/agent-release-notes/java-release-notes/)). If forwarding is disabled, you can use this configuration:
```
